cmd/oras/root/manifest/fetch.go

```diff
@@ -91,15 +91,19 @@ Example - Fetch raw manifest from an OCI layout archive file 'layout.tar':
 	return cmd
 }
 
-func fetchManifest(ctx context.Context, opts fetchOptions) (fetchErr error) {
+func fetchManifest(ctx context.Context, cmd *cobra.Command, opts fetchOptions) (fetchErr error) {
```

Comment:
`ctx` can be obtained via `cmd.Context()`
```suggestion
func fetchManifest(cmd *cobra.Command, opts fetchOptions) (fetchErr error) {
```

Reply:
removed `ctx`.